test/controllers/project_controller_test.exs

```diff
@@ -106,142 +68,66 @@ defmodule CodeCorps.ProjectControllerTest do
   end
 
   describe "create" do
-    @tag :authenticated
+    @tag authenticated: :admin
```

Comment:
This should just be `@tag :authenticated` if you're relying on `insert(:organization_membership, role: "admin")`, otherwise you're going to defer to the site admin authentication since it takes precedence.